.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1536,6 +1694,21 @@ iam_CreateUser:
   synopsis: create an &IAM; user.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateUserExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateUserAsync
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateUser
```

Comment:
These three examples are almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
Followed your suggestion.